core/remote_state_verifier.go

```diff
@@ -0,0 +1,399 @@
+package core
+
+import (
+	"fmt"
+	"github.com/ethereum/go-ethereum/event"
+	"github.com/ethereum/go-ethereum/metrics"
+	"math/rand"
+	"time"
+
+	lru "github.com/hashicorp/golang-lru"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/ethereum/go-ethereum/log"
+)
+
+const (
+	verifiedCacheSize = 256
+	maxForkHeight     = 11
+
+	// defaultPeerNumber is default number of verify peers
+	defaultPeerNumber = 3
+	// pruneHeightDiff indicates that if the height difference between current block and task's
+	// corresponding block is larger than it, the task should be pruned.
+	pruneHeightDiff = 15
+	pruneInterval   = 5 * time.Second
+	resendInterval  = 2 * time.Second
+	// tryAllPeersTime is the time that a block has not been verified and then try all the valid verify peers.
+	tryAllPeersTime = 15 * time.Second
+)
+
+var (
+	remoteVerifyTaskCounter      = metrics.NewRegisteredCounter("remote/state/verify/task/total", nil)
```

Comment:
The layout of these metrics are not proper, please fix them.

Reply:
fixed